composer.json

```diff
@@ -15,7 +15,10 @@
         "php-http/message": "^1.6",
         "php-http/message-factory": "^1.0",
         "phpro/soap-client": "^0.7.11",
-        "psr/http-message": "^1.0"
+        "psr/http-message": "^1.0",
+        "symfony/property-access": "^4.2",
+        "symfony/property-info": "^4.2",
+        "symfony/serializer": "^4.2"
```

Comment:
This would not work with Drupal 8 since Drupal 8 is still on 3.x, we should have something like `"^3|^4"`.

Reply:
Very good point.

Reply:
Fixed.